src/test/java/com/igridin/desktop_calculator/DesktopCalculatorTest.java

```diff
@@ -0,0 +1,38 @@
+package com.igridin.desktop_calculator;
```

Comment:
This is a maven-generated test file. It is a good practice to cover your functionality with tests for several reasons:

0. You can better understand how your code works and whether the design needs to be improved;
0. You can quickly make sure your old code still works while you are adding new functionality;
0. You can refactor the code with less fear of introducing issues;
0. You can be sure everything is in order when other team members update the code;